src/DynamoSandbox/SettingsMigrationWindow.xaml

```diff
@@ -0,0 +1,59 @@
+﻿<Window x:Class="Dynamo.DynamoSandbox.SettingsMigrationWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:ui="clr-namespace:Dynamo.UI;assembly=DynamoCoreWpf"
+        WindowStartupLocation="CenterScreen"
+        WindowStyle="None"
+        ResizeMode="NoResize"
+        Title="First Use Setup"
+        Height="200"
+        Width="300"
+        Style="{DynamicResource DynamoWindowStyle}">
+
+    <Window.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoModernDictionaryUri}" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Window.Resources>
+
+    <Grid>
+        <Grid Column="0"
+              Background="#363636">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="160"></RowDefinition>
+                <RowDefinition Height="40"></RowDefinition>
+            </Grid.RowDefinitions>
+            <Image Grid.Row="0"
+                   Source="/DynamoCoreWpf;component/UI/Images/AboutWindow/logo_about.png"
+                   Width="128"
+                   RenderTransformOrigin="0.531,-0.086"
+                   Margin="86,4,86,25" />
+            <TextBox Grid.Row="0"
+                     Background="Transparent"
+                     Foreground="#888888"
+                     BorderThickness="0"
+                     FontSize="15"
+                     FontFamily="{StaticResource OpenSansRegular}"
+                     Text="First Use Setup"
+                     IsReadOnly="True"
+                     IsReadOnlyCaretVisible="False"
+                     HorizontalAlignment="Center"
+                     VerticalAlignment="Center"
+                     Margin="95,115,95,25" />
+            <TextBox Grid.Row="1"
+                     Background="Transparent"
+                     Foreground="#888888"
+                     BorderThickness="0"
+                     FontSize="14"
+                     FontFamily="{StaticResource OpenSansRegular}"
+                     Text="Migrating Settings ..."
```

Comment:
`Text` elements in this file need to be made localizable (i.e. having `*.resx` files).